osmaxx/excerptexport/models/export.py

```diff
@@ -47,16 +51,23 @@ def get_full_status_update_uri(self, request):
     def status_update_url(self):
         return reverse('job_progress:tracker', kwargs=dict(export_id=self.id))
 
-    def set_and_handle_new_status(self, new_status):
+    def set_and_handle_new_status(self, new_status, *, incoming_request):
         if self.status != new_status:
```

Comment:
I know this is not part of this pull-request: what happens, if the passed in status is `None`? (It will fail, since null=False). This will make all subsequent calls django fail because of our middleware calling this function. Should that be filed as a separate bug?